common/containers/Tabs/ENS/components/NameResolve/components/CountDown.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import moment from 'moment';
+
+class CountDown extends Component<
+  {
+    initialTime: number;
+  },
+  {
+    currentTime: number;
+  }
+> {
```

Comment:
Would prefer keeping the habit of defining a `Props` type rather than inline definitions.

Reply:
I was contemplating between the two formats for this, I'll switch it back